Products/CMFPlone/static/patterns/toolbar/src/toolbar.js

```diff
@@ -390,7 +390,7 @@ define([
           url: $('body').attr('data-portal-url') + path + '/@@render-toolbar'
         }).done(function(data) {
           var $el = $(utils.parseBodyTag(data));
-          that.$el.replaceWith($el);
+          that.$el.parent.replaceWith($el);
```

Comment:
That broke toolbar reloading when browsing folder_contents.
That also shows the importance of having tests, which we apparently do not have here.